gotomagic/handlers.py

```diff
@@ -34,6 +34,9 @@ def open_sublime(code):
     "hack"
     subprocess.call('subl "%s"' % code, shell=True)
 
+def open_vscode(code):
```

Comment:
Sure, lets add this.

We have plans for rewiring goto into a plugin based model. 
In the future These custom commands would be cleaner to implement as a plugin. 
For now it is only good  to add vscode this way. Maybe we should add atom and vim + emacs too?